Artesian/Artesian.SDK.Tests/ActualTimeSerieQueries.cs

```diff
@@ -482,5 +482,118 @@ public void ActWithHeaders_FilterId()
+                 .SetQueryParam("id", 100000001)
+                 .SetQueryParam("fillerK", FillerKind.Null))
+                 .WithVerb(HttpMethod.Get)
+                 .Times(1);
+            }
+        }
+
+        [Test]
+        public void FillerLatestValueActInAbsoluteDateRangeMostRecent()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                var qs = new QueryService(_cfg);
+
+                var act = qs.CreateActual()
+                        .ForMarketData(new int[] { 100000001 })
+                        .InGranularity(Granularity.Day)
+                        .InAbsoluteDateRange(new LocalDate(2018, 6, 22), new LocalDate(2018, 7, 23))
+                        .WithFillLatestValue(Period.FromDays(14))
+                        .ExecuteAsync().Result;
+
+                httpTest.ShouldHaveCalled($"{_cfg.BaseAddress}query/v1.0/ts/Day/2018-06-22/2018-07-23"
+                 .SetQueryParam("id", 100000001)
+                 .SetQueryParam("fillerK", FillerKind.LatestValidValue)
+                 .SetQueryParam("fillerP", "P14D"))
+                 .WithVerb(HttpMethod.Get)
+                 .Times(1);
+            }
+        }
+
+        [Test]
+        public void FillerCustomValueActInAbsoluteDateRangeMostRecent()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                var qs = new QueryService(_cfg);
+
+                var act = qs.CreateActual()
+                        .ForMarketData(new int[] { 100000001 })
+                        .InGranularity(Granularity.Day)
+                        .InAbsoluteDateRange(new LocalDate(2018, 6, 22), new LocalDate(2018, 7, 23))
+                        .WithFillCustom(123)
+                        .ExecuteAsync().Result;
+
+                httpTest.ShouldHaveCalled($"{_cfg.BaseAddress}query/v1.0/ts/Day/2018-06-22/2018-07-23"
+                 .SetQueryParam("id", 100000001)
+                 .SetQueryParam("fillerK", FillerKind.CustomValue)
+                 .SetQueryParam("fillerDV", 123))
+                 .WithVerb(HttpMethod.Get)
+                 .Times(1);
+            }
+        }
+
+        [Test]
+        public void FillerDefaultActInAbsoluteDateRangeMostRecent()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                var qs = new QueryService(_cfg);
+
+                var act = qs.CreateActual()
+                        .ForMarketData(new int[] { 100000001 })
+                        .InGranularity(Granularity.Day)
+                        .InAbsoluteDateRange(new LocalDate(2018, 6, 22), new LocalDate(2018, 7, 23))
+                        .ExecuteAsync().Result;
+
+                httpTest.ShouldHaveCalled($"{_cfg.BaseAddress}query/v1.0/ts/Day/2018-06-22/2018-07-23"
+                 .SetQueryParam("id", 100000001)
+                 .SetQueryParam("fillerK", FillerKind.Null))
```

Comment:
This is not expected. 
Either fillerK is Default or is omitted (as is now).
Which strategy is the Default is implemented server-side, not client side.